delivery_postlogistics/views/stock.xml

```diff
@@ -6,6 +6,16 @@
         <field name="model">stock.picking</field>
         <field name="inherit_id" ref="delivery.view_picking_withcarrier_out_form" />
         <field name="arch" type="xml">
+            <field name="state" position="before">
+                <button
+                    name="action_generate_carrier_label"
+                    help="Create Shipping Label 🚚"
+                    attrs="{'invisible': [('state', '!=', 'done')]}"
```

Comment:
you also have to hide the button when `carrier_id.delivery_type != 'postlogistics'`

Reply:
hello @mmequignon, i updated this domain